src/diffusers/pipelines/flux/pipeline_flux_controlnet.py

```diff
@@ -748,9 +748,11 @@ def __call__(
             )
 
             # set control mode
+            orig_mode_type = type(control_mode)
```

Comment:
if we only accept int for single controlnet, maybe raise a value error here

Reply:
yes good idea

Reply:
There does already appear to be a check for it here inside the controlnet class itself: https://github.com/huggingface/diffusers/blob/b52119ae92067d2a082df558f9eacb583c0860dc/src/diffusers/models/controlnet_flux.py#L289-L290